terraform/environment/region/modules/event_received/lambda.tf

```diff
@@ -7,20 +7,33 @@ module "event_received" {
     GOVUK_NOTIFY_IS_PRODUCTION = data.aws_default_tags.current.tags.environment-name == "production" ? "1" : "0"
     APP_PUBLIC_URL             = "https://${var.app_public_url}"
     UPLOADS_S3_BUCKET_NAME     = var.uploads_bucket.bucket
+    UID_BASE_URL               = var.uid_base_url
   }
-  image_uri   = "${var.lambda_function_image_ecr_url}:${var.lambda_function_image_tag}"
-  ecr_arn     = var.lambda_function_image_ecr_arn
-  environment = data.aws_default_tags.current.tags.environment-name
-  kms_key     = data.aws_kms_alias.cloudwatch_application_logs_encryption.target_key_arn
-  timeout     = 300
-  memory      = 1024
+  image_uri           = "${var.lambda_function_image_ecr_url}:${var.lambda_function_image_tag}"
+  ecr_arn             = var.lambda_function_image_ecr_arn
+  environment         = data.aws_default_tags.current.tags.environment-name
+  kms_key             = data.aws_kms_alias.cloudwatch_application_logs_encryption.target_key_arn
+  iam_policy_document = data.aws_iam_policy_document.api_access_policy.json
```

Comment:
I think this would become an iam policy document override variable that is a list if the other comments are accepted